### ISSUE TITLE
Consider Essential Addons upgrade

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeUKGenealogyWP#168]
Cost-Benefit analysis required

**Benefits**
Testimonial scrolling
Twitter carousel
Logo carousel

**Cost**
$40 per year

**Comment by PatReynolds** [comment]
What is the accessibility of these features?  I know caroussels can have issues.

**Comment by DeniseColbert** [comment]
Good point, will investigate. The main reason we are considering it is for sidebar widget, which would be worth it by itself if it does what we need it to do.

**Comment by johnshaw1000** [comment]
We've found another way for the side bar (at least for now), so shall we reject adding this widget for now?

**Comment by DeniseColbert** [comment]
Moved to icebox and removed high priority tag.
